sapi/fpm/tests/fcgi.inc

```diff
@@ -127,16 +444,24 @@ class Client
      */
     private $_readWriteTimeout = 5000;
 
+    private Transport $transport;
+
     /**
      * Constructor
      *
      * @param string $host Host of the FastCGI application
      * @param int $port Port of the FastCGI application
+     * @param string $transport Transport
      */
-    public function __construct($host, $port)
+    public function __construct($host, $port, string $transport = 'stream')
```

Comment:
Shouldn't we pass the instance directly rather than a key?

Reply:
yeah makes more sense in fcgi part - string match should be more in tester. Will change it.